src/menu/styled-components.js

```diff
@@ -36,6 +42,7 @@ export const ListItem = styled(
     display: 'block',
     color: $isHighlighted ? $theme.colors.primary : $theme.colors.black,
     cursor: 'pointer',
+    backgroundColor: $isHighlighted ? $theme.colors.mono300 : 'transparent',
```

Comment:
by default `initial` is transparent, but if user overrides it, this could cause wrong styling
https://www.w3schools.com/cssref/pr_background-color.asp